app/matplotlib.ipynb

```diff
@@ -86,23 +89,23 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 5,
-   "outputs": [],
    "source": [
     "x = np.random.rand(50)\n",
     "y = np.random.rand(50)\n",
     "colors = np.random.rand(50)\n",
     "sizes = 100 * np.random.rand(50)\n",
     "\n",
-    "# write your code here"
+    "plt.scatter(x, y, c=colors, s=sizes)\n",
+    "plt.title(\"Population over year\")\n",
+    "plt.xlabel(\"Year\")\n",
+    "plt.ylabel(\"Population\")\n",
```

Comment:
The titles and labels for the scatter plot are not descriptive. Consider using titles and labels that are more relevant to the data being visualized, such as 'Scatter Plot of Random Data' for the title, and 'X-axis' and 'Y-axis' for the labels.